src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+const SET_CATEGORY = 'SET_CATEGORY';
+
+const initialState = [];
+
+export const setCategory = (payload) => ({
+  type: SET_CATEGORY,
+  payload
+});
+
+const categoryReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case SET_CATEGORY: {
+      return [...state, action.payload];
```

Comment:
- [ ] This case scenario should simply return the plain text: **'Under construction'**, not an array. Kindly refactor accordingly.